lib/merge_hris_client/api/time_off_balance_api.rb

```diff
@@ -1,196 +0,0 @@
-=begin
-#Merge HRIS API
-
-#The unified API for building rich integrations with multiple HR Information System platforms.
-
-The version of the OpenAPI document: 1.0
-Contact: hello@merge.dev
-Generated by: https://openapi-generator.tech
-OpenAPI Generator version: 5.1.1
-
-=end
-
-require 'cgi'
-
```

Comment:
why is this deleted?

Reply:
time_off_balances_api now exists after the endpoint was renamed